src/kubernetes-ingress-controller/guides/using-kongplugin-resource.md

```diff
@@ -429,34 +521,49 @@ Note the annotation being added to the `KongConsumer` resource.
 Now, if the request is made as the `harry` consumer, the client
 will be rate-limited differently:
 
-```bash
-$ curl -I $PROXY_IP/foo -H 'apikey: my-sooper-secret-key'
+```sh
+curl -I $PROXY_IP/foo -H 'apikey: my-sooper-secret-key'
+```
+
+```
 HTTP/1.1 200 OK
 Content-Type: text/html; charset=utf-8
 Content-Length: 9593
 Connection: keep-alive
+X-RateLimit-Remaining-Minute: 9
+X-RateLimit-Limit-Minute: 10
+RateLimit-Remaining: 9
+RateLimit-Reset: 42
+RateLimit-Limit: 10
 Server: gunicorn/19.9.0
 Access-Control-Allow-Origin: *
 Access-Control-Allow-Credentials: true
-X-RateLimit-Limit-minute: 10
-X-RateLimit-Remaining-minute: 9
 demo:  injected-by-kong
-X-Kong-Upstream-Latency: 3
+X-Kong-Upstream-Latency: 2
 X-Kong-Proxy-Latency: 1
-Via: kong/1.2.1
+Via: kong/2.8.1
+```
+
+And a regular unauthenticated request
 
-# a regular unauthenticated request
-$ curl -I $PROXY_IP/bar
+```sh
+curl -I $PROXY_IP/bar
+```
+
+```sh
 HTTP/1.1 200 OK
 Content-Type: text/plain; charset=UTF-8
 Connection: keep-alive
+X-RateLimit-Remaining-Minute: 4
+X-RateLimit-Limit-Minute: 5
+RateLimit-Remaining: 4
+RateLimit-Reset: 11
+RateLimit-Limit: 5
 Server: echoserver
-X-RateLimit-Limit-minute: 5
-X-RateLimit-Remaining-minute: 4
 demo:  injected-by-kong
-X-Kong-Upstream-Latency: 2
+X-Kong-Upstream-Latency: 1
 X-Kong-Proxy-Latency: 1
-Via: kong/1.2.1
+Via: kong/2.8.1
 ```
 
 This guide demonstrates how you can use the {{site.kic_product_name}} to
```

Comment:
This last statement seems super out of place at the end of the page. I know this isn't part of what you were fixing, but can you move it to the start of the topic instead? Otherwise, I'm not sure we'll ever remember to come back and fix it.

Reply:
Thanks for your help @lena-larionova 😄 , please check the latest where I modified the outro a bit, and I moved some of this information to the intro and tried to simplify and clarify it.

Also, I added a link to Plugin Hub, if you wouldn't mind double checking how I referenced that in the outro / CTA here on this page.  👍